packages/faceted-search/src/queryClient/index.js

```diff
@@ -0,0 +1,4 @@
+import { createTqlQuery } from './tql';
+
+// eslint-disable-next-line import/prefer-default-export
+export { createTqlQuery };
```

Comment:
This is what we import in our app to convert badges to tql?

Reply:
Yep it's the lonk between badges and query language